docs/04-operation-guides/troubleshooting/apix-03-dns-mgt-could-not-resolve-host.md

```diff
@@ -16,7 +16,7 @@ The error could result from:
 
 - Timing issues during the DNS Entry creation
 - VPN connection
-- Invalid DNS settings on your OS.
+- Invalid DNS settings on your OS
```

Comment:
In the "Remedy" section (I can't comment there):
- Add info that DNSEntry status should be: `Ready`
- Add a point: "Check if your local DNS configuration in /etc/hosts (or an equivalent file on your OS) contains an entry for the target host. Remove it."